.github/workflows/release.yml

```diff
@@ -0,0 +1,98 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      qemu_repo:
+        description: 'QEMU repo (e.g. https://github.com/qemu/qemu)'
+        required: false
+      qemu_version:
```

Comment:
how would this do a current buildkit release where source is set with a sha?

Reply:
`target` input handle the bake target to run. If you set `qemu_version` with a sha the git/docker tag will be the sha commit (maybe we could use a short tag if a long one is provided). That's why I thought it would be better to extract the internal [version available in qemu repo](https://github.com/qemu/qemu/blob/master/VERSION) to create proper docker/git tag.

Reply:
Why not just input for the commit tag. That default to source ref (or to not tagging at all)

Reply:
I guess we could find the most recent tag in the tree from the sha and prepend its value. e.g. `v5.2.0-be25039`, WDYT?

Reply:
I don't think we need lots of magic autodetection. This will be used infrequently by a small number of people. They can give the correct inputs, just needs to be a way to enter it.